charmander/routes/example/controller.py

```diff
@@ -0,0 +1,21 @@
+"""
+Example controller.
+
+"""
+from microcosm.api import binding
+from microcosm_flask.conventions.crud_adapter import CRUDStoreAdapter
+from microcosm_flask.namespaces import Namespace
+
+from charmander.models.example_model import Example
+
+
+@binding("example_controller")
+class ExampleController(CRUDStoreAdapter):
```

Comment:
Most of the logic exists in the default `CRUDStoreAdapter` from `microcosm-flask`. To get basic `create`, `retrieve`, `update`, `delete` you can defer to the defaults from the base class. Customization can go here when it is needed.